.github/workflows/trackdeploys.yaml

```diff
@@ -0,0 +1,55 @@
+name: Track semVer through environments
+
+on:
+  schedule:
+    - cron: '0 */3 * * *' # run every 3 hours and on push
+  workflow_dispatch: {}
+
+jobs:
+  get_versions:
+    name: Get semVer for environments
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_BUCKET_SA_KEY }}
+          export_default_credentials: true
+      - name: Use gcloud copy endpoints.json
+        run: gsutil cp gs://deployment-tracking/endpoints.json .
+      - shell: bash
+        run: |
+          endPoints+=( "https://jade.datarepo-dev.broadinstitute.org/configuration" \
+            "https://data.alpha.envs-terra.bio/configuration" \
+            "https://data.staging.envs-terra.bio/configuration" \
+            "https://data.terra.bio/configuration")
+
+          for i in ${endPoints[@]}; do
+            if [[ "$i" == *"dev"* ]]
+            then
+             env=dev
+            elif [[ "$i" == *"alpha"* ]]
+            then
+             env=alpha
+            elif [[ "$i" == *"staging"* ]]
+            then
+             env=staging
+            elif [[ "$i" == *"data.terra.bio"* ]]
+            then
+             env=production
+            else
+             echo "No env found"
+            fi
+            date=$(date +%Y-%m-%d\ %H:%M:%S)
```

Comment:
This way, the dates for each release may be slightly different. Pulling `date` outside the `for` will give you the same date for all of them.

Reply:
probably a good idae

Reply:
date moved out of for loop